{
  "PublicDateAtUSN": "0001-01-01T00:00:00Z",
  "CRD": "0001-01-01T00:00:00Z",
  "Candidate": "CVE-2024-5629",
  "PublicDate": "2024-06-05T15:15:00Z",
  "References": [
    "https://www.cve.org/CVERecord?id=CVE-2024-5629",
    "https://jira.mongodb.org/browse/PYTHON-4305"
  ],
  "Description": "An out-of-bounds read in the 'bson' module of PyMongo 4.6.2 or earlier allows deserialization of malformed BSON provided by a Server to raise an exception which may contain arbitrary application memory.",
  "UbuntuDescription": "",
  "Notes": null,
  "Bugs": null,
  "Priority": "medium",
  "DiscoveredBy": "",
  "AssignedTo": "leosilva",
  "Patches": {
    "pymongo": {
      "devel": {
        "Status": "needs-triage",
        "Note": ""
      },
      "esm-infra/bionic": {
        "Status": "needs-triage",
        "Note": ""
      },
      "esm-infra/xenial": {
        "Status": "needs-triage",
        "Note": ""
      },
      "focal": {
        "Status": "needs-triage",
        "Note": ""
      },
      "jammy": {
        "Status": "needs-triage",
        "Note": ""
      },
      "mantic": {
        "Status": "needs-triage",
        "Note": ""
      },
      "noble": {
        "Status": "needs-triage",
        "Note": ""
      },
      "trusty/esm": {
        "Status": "needs-triage",
        "Note": ""
      },
      "upstream": {
        "Status": "needs-triage",
        "Note": ""
      }
    }
  },
  "UpstreamLinks": {
    "pymongo": [
      "https://github.com/mongodb/mongo-python-driver/commit/56b6b6dbc267d365d97c037082369dabf37405d2 (v4.6)",
      "https://github.com/mongodb/mongo-python-driver/commit/372b5d68d5a57ccc43b33407cd23f0bc79d99283 (v4.8)"
    ]
  }
}